src/deep-blue-green/backend/src/replication/start/bootstrap.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * Created by CCristi on 2/10/17.
+ */
+
+/* eslint no-use-before-define:0 */
+
+'use strict';
+
+const AWS = require('aws-sdk');
+const CHUNK_SIZE = 100;
+
+exports.handler = (event, context, callback) => {
+  const tableManager = new TableManager(new AWS.DynamoDB(), event.Table);
+  const backFillExecutor = new BackFillExecutor(tableManager, event.Hash);
+  const scanPayload = {};
+
+  scanPayload[':hash'] = {
+    S: event.Hash,
+  };
+
+  const backFillProcessor = () => {
+    tableManager
+      .scan(['Id'], TableManager.BACKFILL_FILTER, scanPayload)
+      .then(result => {
+        let itemChunks = result.Items
+          .map(i => new Item(i))
+          .reduce(arrayChunkReducer(CHUNK_SIZE), []);
+
+        return backFillExecutor.backFillChunks(itemChunks).then(() => result.length);
+      })
+      .then(itemsProcessed => {
+        if (itemsProcessed > 0) {
+          console.log(`"${itemsProcessed}" items have been marked`);
+          backFillProcessor();
+        } else {
+          callback(null, {});
+        }
+      })
+      .catch(e => {
+        callback({
```

Comment:
Please follow the following standard for callbacks: `callback(error || null, null || result)` where `error instanceof Error`